src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -205,9 +208,36 @@ stop_metadata_sync_to_node(PG_FUNCTION_ARGS)
 						errmsg("node (%s,%d) does not exist", nodeNameString, nodePort)));
```

Comment:
`FindWorkerNode` doesn't include the replicas/secondary nodes. So, if I try to stop metadata sync on a replica node, I get an unexpected error:
```SQL
select stop_metadata_sync_to_node('localhost', 5505);
ERROR:  node (localhost,5505) does not exist
Time: 0.191 ms

```

Where, the node actually exists:
```SQL
table pg_dist_node;
 nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole  | nodecluster | metadatasynced | shouldhaveshards 
--------+---------+-----------+----------+----------+-------------+----------+-----------+-------------+----------------+------------------
      2 |       1 | localhost |     5506 | default  | f           | t        | secondary | cluster_c   | f              | t
      3 |       1 | localhost |     5505 | default  | f           | t        | secondary | cluster_b   | f              | t
      1 |       1 | localhost |     5504 | default  | t           | t        | primary   | default     | t              | t
(3 rows)

```


I think we should use `FindWorkerNodeAnyCluster` which would get the secondaries as well